crates/ruff/src/args.rs

```diff
@@ -1255,22 +1232,10 @@ impl ConfigurationTransformer for ExplicitConfigOverrides {
         }
         config.lint.rule_selections.push(RuleSelection {
             select: self.select.clone(),
-            ignore: self
-                .ignore
-                .iter()
-                .cloned()
-                .chain(self.extend_ignore.iter().cloned())
-                .flatten()
-                .collect(),
```

Comment:
This is a bit unfortunate. We never added deprecation warnings to the CLI. I'm inclined to keep the CLI options for now but add a deprecation warning instead.